addons/wowl/static/src/components/dropdown/dropdown_item.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from "@odoo/owl";
+
+export class DropdownItem extends Component {
+  static template = "wowl.DropdownItem";
+  static props = {
+    value: {
```

Comment:
I had some shower thoughts: We know that value or data or payload is what should be returned on item click. But to the user, he's passing a prop to an <DropdownItem> that's called value. He may even think that it acts a bit like a t-esc or something. Our prop name doesn't explicitly its intent at all. We should call it a little more verbosely, but with reasons. Something like: onClickPayload or clickPayload.

Reply:
Let's go with the 'onClickPayload' version